sdk/search/Azure.Search.Documents/src/Generated/Models/EdgeNGramTokenFilter.Serialization.cs

```diff
@@ -12,79 +12,5 @@ namespace Azure.Search.Documents.Indexes.Models
 {
     public partial class EdgeNGramTokenFilter : IUtf8JsonSerializable
     {
-        void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
```

Comment:
Did this change at all?  Could we leave the generated version?

Reply:
Unfortunately, no. Merging types is not supported by the generator, and if I don't completely take over de/serialization all sorts of bad things happen that make the source invalid and the project unable to compile.